examples/next/docs/js/accessibility/README.md

```diff
@@ -0,0 +1,25 @@
+# Accessibility Demo
+
+This demo aims to showcase the accessibility features of Handsontable when used in a Javascript application. It demonstrates the configurability of Handsontable from an accessibility perspective.
+
+## How to run this example
+### Installation
+
+Call `npm install` to install all necessary dependencies.
+
+### Development
+
+To start local development server call `npm run start`. Now you can visit http://localhost:5173 to view this project.
+
+## License
+
+Handsontable is a commercial software with two licenses available:
+
+- Free for non-commercial purposes such as teaching, academic research, and evaluation. [Read it here](https://github.com/handsontable/handsontable/blob/master/handsontable-non-commercial-license.pdf).
+- Commercial license with support and maintenance included. See [pricing plans](https://handsontable.com/pricing).
+
+## Technical support
+
+If you have a commercial license and your support plan is active, contact our [Technical Support Team](https://handsontable.com/contact?category=technical_support).
+
+We also support free-tier users through [GitHub issues](https://github.com/handsontable/handsontable/issues).
```

Comment:
I would change this sentence into : "We also provide support for free-tier users through"